web/src/beta/lib/lexical/RichTextEditor/plugins/ToolbarPlugin.tsx

```diff
@@ -90,6 +90,20 @@ const FONT_SIZE_OPTIONS: [string, string][] = [
   ["20px", "20px"],
 ];
 
+const LINE_HEIGHT_OPTIONS: [string, string][] = [
```

Comment:
Don't see why you don't just do `["1.0", "1.1", etc, etc]`?
Is it just in case we sometime want to display some different value to the user? or..?

Reply:
I feel the same...As you know this plugin is basically referenced from the official demo and i guess the origin author did this for some reason so i kept the same.